fv3core/fv3core/stencils/fxadv.py

```diff
@@ -655,6 +642,8 @@ def __call__(
             vc_contra,
             dt,
         )
+        # self._set_nans(uc_contra)
+        # self._set_nans(vc_contra)
```

Comment:
Because this set_nans call is only useful in translate tests that are larger in scope than fxadv, like FVDynamics. 

The fxadv translate class will strip these nans off when testing. If I put this in TranslateFxAdv, the code would look like:
```
uc_contra = set_nans(uc_contra)  # sets nans in outermost halo
uc_contra = subselect(uc_contra)  # excludes outermost halo
```
in that case, setting nans carries no meaning because they're immediately discarded.

We want to see if these nans propagate into compute-domain information in one of the higher-level classes that rely on fxadv, like TranslateFVDynamics. In TranslateFVDynamics, we never see uc_contra from inside the translate class, so there's no opportunity to set nans in the center of the execution apart from dynamically modifying the class behavior (which dace doesn't like) or having code inside the compute class (which I did here).